client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -60,11 +60,14 @@ class VerificationPortalForm extends Component {
 				/>
 				<div className={classes.formSection}>
 					<div className={classes.gridField}>
-						<div className={classes.firstBox}>Verified</div>
-						<div className={classes.secondAndThirdBox}>
-							<Heading text="GENERAL INFORMATION" />
+						<div className={classes.firstBox}>
+							<Heading text="VERIFIED" size="small" />
+						</div>
+						<div className={classes.headingGrid}>
+							<Heading text="GENERAL INFORMATION" size="medium" />
 						</div>
 					</div>
+					<Spacer height="15px" />
```

Comment:
This space is 55px

<img width="395" alt="screen shot 2018-07-03 at 2 38 59 pm" src="https://user-images.githubusercontent.com/6378435/42241072-ef64d080-7ece-11e8-954e-cb75f924c5b3.png">